shared/macros-bash.jinja

```diff
@@ -156,6 +156,170 @@ fi
 dconf update
 {{%- endmacro -%}}
 
+{{#
+# Macro to enable or disable a particular service.
+#
+# Examples:
+#
+# bash_service_command("enable", "bluetooth")
+# bash_service_command("disable", "bluetooth.service")
+# bash_service_command("disable", "rsh.socket", xinetd="rsh")
+#}}
+{{%- macro bash_service_command(service_state, service, xinetd="") -%}}
+local service_state="{{{ service_state }}}"
+local service="{{{ service }}}"
+local xinetd="{{{ xinetd }}}"
+
+# If systemctl is installed, use systemctl command; otherwise, use the service/chkconfig commands
```

Comment:
At compile-time, we know what init system the concerned product uses, so you can go more the `bash_package_install` way, where the expanded macro is super simple.